src/test/ui/tool-attributes-misplaced-1.rs

```diff
@@ -8,21 +8,21 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
-#![feature(tool_attributes)]
+#![feature(tool_attributes, custom_attribute)]
```

Comment:
Oh one thing I was wondering, this feature was just added to test that `#[rustfmt]`, typically illegal under `#![feature(tool_attributes)]`, is in fact actually allowed with `#![feature(custom_attributes)]`?

Reply:
Yes, `rustfmt` resolves to a tool module in type namespace, but it resolves to nothing in macro namespace and is thus interpreted as a custom attribute.

Unfortunately, feature gate errors are fatal and stop compilation for some reason, so I had to enable the feature to get other errors, which are reported later.
I should fix this some day.